t/selenium/release-editor/Seeding.json5

```diff
@@ -289,17 +289,17 @@
     {
       command: 'assertEval',
       target: "Array.from(document.querySelectorAll('#external-links-editor tr[class=\"external-link-item\"]')).slice(0,3).map(x => x.querySelector('a.url') ? x.querySelector('a.url').href : x.querySelector('input').value).join('\\n')",
-      value: 'http://foo.bar.baz/\n\nhttp://foo.bar.baz/foo/',
+      value: 'http://foo.bar.baz/\nhttp://www.amazon.co.jp/\n',
```

Comment:
Did you replace this example to trigger a different error message? Just asking because I’m not sure.

Reply:
No, it's just because there's no ASIN option after restricting type options, it'll only appear when an Amazon link is entered.